src/redux/books/books.js

```diff
@@ -6,11 +6,19 @@ const REMOVE_BOOK = 'REMOVE_BOOK';
 const books = (state = [], action) => {
   switch (action.type) {
     case ADD_BOOK:
-      return action.book;
-
+      return [
+        ...state,
+        {
+          added: true,
+        },
+      ];
     case REMOVE_BOOK:
-      return action.book;
-
+      return [
+        ...state,
+        {
+          added: false,
+        },
+      ];
```

Comment:
- [ ] in case `action.type` is `REMOVE_BOOK` the function use ES6 filter() method to create a new array, which will not contain the book you want to remove from the store (filter by the id key - i.e.: return state.filter(book => book.id !== id)